examples/resources/kentik-synthetic_test-bulk-create/provider.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    kentik-synthetics = {
+      version = "~> 0.1.0"
+      source  = "kentik.com/automation/kentik-synthetics"
```

```suggestion
      source  = "kentik/automation/kentik-synthetics"
```
Please use that for simplicity. I do not remember exact use case, but it was also modified in cloudexport provider in such way and it's better to have it consistent.

Other occurences modified here: https://github.com/kentik/terraform-provider-kentik-synthetics/pull/26/files

---

I'm getting this error while changing the source attribute:
> │ Error: Failed to query available provider packages
> │ 
> │ Could not retrieve the list of available versions for provider kentik/automation/kentik-synthetics: could not connect to > > kentik: Failed to request discovery document: Get
> │ "https://kentik/.well-known/terraform.json": dial tcp: lookup kentik on 127.0.0.53:53: server misbehaving
> 
> Error. Exit 1

---

will be possible to change as you adviced after this pull request is merged: https://github.com/kentik/terraform-provider-kentik-synthetics/pull/26

---

This is unfortunate behavior of Terraform SDK. Unless you have locally checked out source code for the provider or the provider is published in the registry, the SDK interprets the first path element as a hostname and tries to do DNS resolution on it, which though should silently fail. @ziemowit141, your DNS resolver is probably misconfigured. Do you really have DNS server listening on 127.0.0.53 port 53? In any case, we do not want to have `kentik.com` in the source attribute, because that results in always failing unnecessary requests to our main webserver. Kentik is unlikely to ever host Terraform registry at `kentik.com` :).